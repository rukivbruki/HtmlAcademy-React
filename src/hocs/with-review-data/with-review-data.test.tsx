import * as React from "react";
import * as renderer from "react-test-renderer";
import withReviewData from "./with-review-data";

const MockComponent = () => <div>Test</div>;

const MockComponentWrapped = withReviewData(MockComponent);

it(`withReviewData renders correctly`, () => {
  const tree = renderer.create((
    <MockComponentWrapped />
  ), {
    createNodeMock() {
      return {};
    }
  }).toJSON();

  expect(tree).toMatchSnapshot();
});
