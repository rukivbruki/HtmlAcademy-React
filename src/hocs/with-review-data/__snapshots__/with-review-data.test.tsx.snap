// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`withReviewData renders correctly 1`] = `
<div>
  Test
</div>
`;
